protobuf/daemon/status.proto

```diff
@@ -6,6 +6,20 @@ option go_package = "github.com/NordSecurity/nordvpn-linux/daemon/pb";
 
 import "config/protocol.proto";
 import "config/technology.proto";
+import "config/group.proto";
+
+enum ConnectionSource {
```

Comment:
Is there defined/agreed a purpose of connection source? It is not being logged, so maybe a frontend is planning to show it to the user?

Reply:
Yeah, it is a GUI requirement.